installer/packer/scripts/package_provisioning.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # Install sudo and ESX-optimized kernel
-tdnf install -y sudo linux-esx rsync lvm2 docker gawk parted tar
+tdnf install -y sudo linux-esx rsync lvm2 docker gawk parted tar python2
```

Comment:
all the prereqs for the configuration are already satisfied, `python2` is not needed to run `harbor/prepare`